content/08-cardano-sidechains/03-example-evm-sidechain/06-transferring-tokens.mdx

```diff
@@ -0,0 +1,222 @@
+
+```shell
+curl -L -X POST -H 'Content-type:application/json' -d '{
+  "jsonrpc": "2.0",
+  "method": "sidechain_getOutgoingTxMerkleProof",
+  "params": [<N or N+1>, <txIndex>],
+  "id": 1
+  }' https://faucet.sidechain.evmtestnet.iohkdev.io/ | jq
+```
+
+Example: 
+
+```
+curl -L -X POST https://faucet.sidechain.evmtestnet.iohkdev.io/ -H 'Content-type:application/json' -d '{"jsonrpc": "2.0", "method": "sidechain_getOutgoingTxMerkleProof", "params": [1295, 1], "id": 1}' | jq
+
+{
+  "jsonrpc": "2.0",
+  "result": {
+    "proof": {
+      "bytes": "0xd8799fd8799f0101581d6045736b33c53d9f526a223086afd812b07ed9741da80e989e96998893d8799f5820772485d60f6744cf252f26560413aae8d28c82a88b1c77eede792f28965f4e79ffff9fd8799f005820ed69142610619b748ec5cd657e418c1c891c3a176900376d12db0b3c406a0a38ffffff",
+      "info": {
+        "transaction": {
+          "value": "0x1",
+          "recipient": "0x6045736b33c53d9f526a223086afd812b07ed9741da80e989e96998893",
+          "txIndex": 1
+        },
+        "merkleRootHash": "0xaacd27079b9c8b8bd17693beccb8997b4b7b1ef97596018d43047a40dab6b021"
+      }
+    },
+    "sidechainEpoch": 1295
+  },
+  "id": 1
+}
+```
+
+From the return you need to obtain the `proof/bytes` and remove the `0x` that is prepended. Call that string `merkleProof`. 
+
+*In the above example `d8799fd8799...` is the Merkle proof.*
+
+- Wait until the epoch changes and the committee handover has happened. The committee handover happens automatically on the testnets by a service (relay) that tries every minute to find epoch signatures, and succeeds after a new epoch has started and the signatures are obtained. On mainnet, the committee handover will be done by anyone who wants, where there will be an incentive to do so. A good way to verify this is to wait until you observe your transaction in the list of signatures to upload:
+
+```
+curl -L -X POST -H 'Content-type:application/json' -d '{
+    "jsonrpc": "2.0",
+    "method": "sidechain_getSignaturesToUpload",
+    "params": [<limit>],
+    "id": 1
+    }' https://faucet.sidechain.evmtestnet.iohkdev.io/ | jq
+```
+
+Where `limit` = min(limit, number of epochs that are not yet relayed). The default value is `100`, but you can set it to any other number. Normally, when the handover happens at every epoch, it should only show 0 or 1 epochs (the last one). Once you identify that there are no roothashes pending in the list under the epoch that your lock was processed (`N` or `N+1`), you can claim your tokens. By now the epoch will be `N+1 `or `N+2`, depending on the phase you performed the lock operation in.
+
+#### Step 05 Claim SC_Token
+
+- Claim the tokens by submitting a transaction on the main chain with the `merkleProof` you obtained using the **sidechain-cli**: 
+
+```shell
+./sidechain-cli claim-fuel --mamba-url https://faucet.sidechain.evmtestnet.iohkdev.io/ --signing-key-file /PATH/TO/payment.skey --combined-proof <merkleProof without 0x>
```

Comment:
claim won't work if we don't have config.json, and we don't say anything about it.

Reply:
Also, we need to run that command from `/bin` directory, otherwise we'll get an error (https://input-output.atlassian.net/browse/ETCM-3226). Let's add a note on that.

Reply:
There are 3 mamba-url references, I think this is a release blocker.
Artur and Lech and Radek found these and there is a ticket in progress to replace them, but the docu should be updated too.